docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-opsgenie.md

```diff
@@ -0,0 +1,61 @@
+---
+canonical: https://grafana.com/docs/grafana/latest/alerting/configure-notifications/manage-contact-points/integrations/configure-opsgenie/
+description: Configure the Opsgenie integration to receive notifications when your alerts are firing
+keywords:
+  - grafana
+  - alerting
+  - Opsgenie
+  - integration
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+menuTitle: Opsgenie
+title: Configure Opsgenie for Alerting
+weight: 300
+---
+
+# Configure Opsgenie for Alerting
+
+Use the Grafana Alerting - Opsgenie integration to receive alert notifications in your Opsgenie alert dashboard when your Grafana alert rules are triggered and resolved.
+
+## Before you begin
+
+Create an API key to enable Grafana to send alert notifications to Opsgenie alert dashboard.
```

Comment:
add a space between these two lines - actually wrote this in the other integration doc but it auto-merged before